client/src/AppBundle/Resources/translations/user-activate-expired.en.yml

```diff
@@ -1,5 +1,5 @@
 page:
```

Comment:
I think both the pageTitle and htmlTitle should probably be changed to link expired for this